src/kernels/linux-hardened.sh

```diff
@@ -0,0 +1,109 @@
+kernel_version_max_git="${kernel_version_max}"
+zfs_git_commit=""
+spl_git_commit=""
+zfs_git_url="https://github.com/zfsonlinux/zfs.git"
+spl_git_url="https://github.com/zfsonlinux/spl.git"
+
+header="\
+# Maintainer: Jan Houben <jan@nexttrex.de>
+# Maintainer: Jesus Alvarez <jeezusjr at gmail dot com>
+#
+# This PKGBUILD was generated by the archzfs build scripts located at
+#
+# http://github.com/archzfs/archzfs
+#
+# ! WARNING !
+#
+# The archzfs packages are kernel modules, so these PKGBUILDS will only work with the kernel package they target. In this
+# case, the archzfs-linux-hardened packages will only work with the default linux-hardened package! To have a single PKGBUILD target
+# many kernels would make for a cluttered PKGBUILD!
+#
+# If you have a custom kernel, you will need to change things in the PKGBUILDS. If you would like to have AUR or archzfs repo
+# packages for your favorite kernel package built using the archzfs build tools, submit a request in the Issue tracker on the
+# archzfs github page.
+#
+#"
+
+update_linux_hardened_pkgbuilds() {
+    pkg_list=("spl-linux-hardened" "zfs-linux-hardened")
+    kernel_version_full=$(kernel_version_full ${kernel_version})
+    kernel_version_full_pkgver=$(kernel_version_full_no_hyphen ${kernel_version})
+    kernel_version_major=${kernel_version%-*}
+    kernel_mod_path="extramodules-${kernel_version}-hardened"
+    archzfs_package_group="archzfs-linux-hardened"
+    spl_pkgver=${zol_version}_${kernel_version_full_pkgver}
+    zfs_pkgver=${zol_version}_${kernel_version_full_pkgver}
+    spl_pkgrel=${pkgrel}
+    zfs_pkgrel=${pkgrel}
+    spl_conflicts="'spl-linux-hardened-git'"
+    spl_headers_conflicts="'spl-linux-hardened-git-headers' 'spl-linux-lts-headers' 'spl-linux-lts-git-headers' 'spl-linux-hardened-headers' 'spl-linux-hardened-git-headers'"
+    zfs_headers_conflicts="'zfs-linux-hardened-git-headers' 'zfs-linux-lts-headers' 'zfs-linux-lts-git-headers' 'zfs-linux-headers' 'zfs-linux-git-headers'"
+    zfs_conflicts="'zfs-linux-hardened-git'"
+    spl_utils_pkgname="spl-utils-common"
+    spl_pkgname="spl-linux-hardened"
+    zfs_utils_pkgname="zfs-utils-common"
+    zfs_pkgname="zfs-linux-hardened"
+    spl_pkgbuild_path="packages/${kernel_name}/${spl_pkgname}"
+    zfs_pkgbuild_path="packages/${kernel_name}/${zfs_pkgname}"
+    spl_src_target="https://github.com/zfsonlinux/zfs/releases/download/zfs-${zol_version}/spl-${zol_version}.tar.gz"
+    zfs_src_target="https://github.com/zfsonlinux/zfs/releases/download/zfs-${zol_version}/zfs-${zol_version}.tar.gz"
+    spl_workdir="\${srcdir}/spl-${zol_version}"
+    zfs_workdir="\${srcdir}/zfs-${zol_version}"
+    linux_depends="\"linux-hardened>=${kernel_version}\" \"linux-hardened<${kernel_version_max}\""
+    linux_headers_depends="\"linux-hardened-headers>=${kernel_version}\" \"linux-hardened-headers<${kernel_version_max}\""
+    zfs_makedepends="\"${spl_pkgname}-headers\""
+}
+
+# update_linux_hardened_git_pkgbuilds() {
```

Comment:
I didn't want to submit zfs-linux-hardened and zfs-linux-hardened-git to the aur before knowing, if this would get merged 😄 
You can just uncomment it, and it should work